src/app/components/search-results/search-results.component.ts

```diff
@@ -0,0 +1,36 @@
+import { Component, Input } from '@angular/core';
+import { SortWord } from '../../shared/interfaces';
+import { HttpService } from '../../shared/services/http.service';
+import { Observable } from 'rxjs';
+import { SearchResponse } from '../../shared/models/search-response.model';
```

Comment:
it is better to organize imports:
1. third party import 
empty line
2. own imports